lib/unimatrix/authorization/filters/requires_policies.rb

```diff
@@ -70,29 +69,30 @@ def self.included( controller )
       controller.extend( ClassMethods )
     end
 
-    def policies=(attributes)
+    def policies=( attributes )
       @policies = attributes
     end
 
     def policies
-      @policies ||= begin
```

Comment:
This needs to be put back to work the way it was. Removing this will break archivist policy checks in certain places, where the before action isn't called but we check policies specifically - basically anywhere that uses the "has_permission?" method.